client/cli/tx.go

```diff
@@ -9,21 +9,35 @@ package cli
 import (
 	"fmt"
 
+	"cosmossdk.io/x/upgrade/plan"
+	upgradetypes "cosmossdk.io/x/upgrade/types"
 	"github.com/cosmos/cosmos-sdk/client"
 	"github.com/cosmos/cosmos-sdk/client/flags"
 	"github.com/cosmos/cosmos-sdk/client/tx"
+	sdk "github.com/cosmos/cosmos-sdk/types"
 	authclient "github.com/cosmos/cosmos-sdk/x/auth/client"
+	clienttypes "github.com/cosmos/ibc-go/v8/modules/core/02-client/types"
 	"github.com/noble-assets/authority/types"
 	"github.com/spf13/cobra"
 )
 
+const (
+	FlagUpgradeHeight      = "upgrade-height"
+	FlagUpgradeInfo        = "upgrade-info"
+	FlagNoValidate         = "no-validate"
+	FlagNoChecksumRequired = "no-checksum-required"
+	FlagDaemonName         = "daemon-name"
+)
+
 func GetTxCmd() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   types.ModuleName,
 		Short: fmt.Sprintf("Transactions commands for the %s module", types.ModuleName),
 	}
 
 	cmd.AddCommand(NewCmdExecute())
+	cmd.AddCommand(NewCmdSubmitUpgrade())
+	cmd.AddCommand(newSubmitRecoverClientCmd())
```

Comment:
Could we align these a little more with our other modules? 🙏🏻

```suggestion
	cmd.AddCommand(NewCmdSoftwareUpgrade())
	cmd.AddCommand(NewCmdRecoverClient())
```

Reply:
[`cd24a3d` (#4)](https://github.com/noble-assets/authority/pull/4/commits/cd24a3d9e28383485b3ea38afd49dd96ab90e3a1)